foundation-ui/navigation/NavigationService.js

```diff
@@ -0,0 +1,111 @@
+import {EventEmitter} from 'events';
+import EventTypes from './EventTypes';
```

Comment:
Let's do `import {NAVIGATION_CHANGE} from './EventTypes';`
This pattern is better as an error will occur if we mistype a constant, rather than accidentally fat-fingering it when we access the object eg. `EventTypes.FOo_BAR`